docs/source/guide/guide-getting-started.rst

```diff
@@ -262,3 +266,63 @@ We can then use this backend for our mitigation.
 
 Note that we don't need to even redefine factories for different stacks. Once
 you have a ``Factory`` it can be used with different front and backends.
+
+Error Mitigation with Probabilistic Error Cancellation
+------------------------------------------------------
+
+In *Mitiq*, it is very easy to switch between different error mitigation methods.
+
+For example, we can implement Probabilistic Error Cancellation (PEC) by using the same execution function (``executor``)
+and the same *Cirq* circuit (``circuit``) that we have already defined in the section
+:ref:`Multi-platform Framework <multi_platform_framework>`.
+
+Differently from ZNE, PEC requires the knowledge of the noise model and of the noise strength acting on the system.
```

Comment:
I would say PEC depends on knowlege of the noise model and strength *more than ZNE*.

Reply:
I replaced this with:
`PEC requires a good knowledge of the noise model and of the noise strength acting on the system.`
without any comparison with ZNE.